packages/calcite-components/src/components/input-date-picker/input-date-picker.scss

```diff
@@ -189,3 +208,39 @@
 }
 
 @include base-component();
+
+calcite-date-picker {
+  --calcite-date-picker-border-color: var(--calcite-input-date-picker-border-color);
+  --calcite-date-picker-corner-radius: var(--calcite-input-date-picker-corner-radius);
+  --calcite-date-picker-day-background-color-hover: var(--calcite-input-date-picker-day-background-color-hover);
+  --calcite-date-picker-day-background-color-selected: var(--calcite-input-date-picker-day-background-color-selected);
+  --calcite-date-picker-day-background-color: var(--calcite-input-date-picker-day-background-color);
+  --calcite-date-picker-day-corner-radius: var(--calcite-input-date-picker-day-corner-radius);
+  --calcite-date-picker-day-range-background-color-hover: var(
+    --calcite-input-date-picker-day-range-background-color-hover
+  );
+  --calcite-date-picker-day-range-background-color-selected: var(
+    --calcite-input-date-picker-day-range-background-color-selected
+  );
+  --calcite-date-picker-day-text-color-hover: var(--calcite-input-date-picker-day-text-color-hover);
+  --calcite-date-picker-day-text-color: var(--calcite-input-date-picker-day-text-color);
+  --calcite-date-picker-day-text-selected: var(--calcite-input-date-picker-day-text-selected);
+  --calcite-date-picker-week-headers-text-color: var(--calcite-input-date-picker-week-headers-text-color);
```

Comment:
Should this be `header` (singular) since there's only one week header? Can be tackled in a follow-up PR.